tests/cpu/Config_tests.cpp

```diff
@@ -181,6 +181,39 @@ OCIO_ADD_TEST(Config, simple_config)
     OCIO_CHECK_NO_THROW(config->sanityCheck());
 }
 
+OCIO_ADD_TEST(Config, missing_space_key_value_pair)
+{
+
+    constexpr char SIMPLE_PROFILE[] =
+        "ocio_profile_version: 2\n"
+        "search_path: luts\n"
+        "roles:\n"
+        "  default: raw\n"
+        "file_rules:\n"
+        "  - !<Rule> {name: Default, colorspace: default}\n"
+        "displays:\n"
+        "  Disp1:\n"
+        "    - !<View> {name: View1, colorspace: raw}\n"
+        "active_displays: []\n"
+        "active_views: []\n"
+        "colorspaces:\n"
```

Comment:
This is not a valid ColorSpace definition, which is why your test is failing in CI. Please have a look at the other tests in this file to see what is missing. You can run these tests locally to debug the issue. After you build run: `ctest -V`